examples/logging/example.go

```diff
@@ -12,17 +12,17 @@ import (
 
 var (
 	// global error values can be useful when wrapping errors or inspecting error types
-	ErrInternalServer = eris.New("error internal server")
+	errInternalServer = eris.New("error internal server")
```

Comment:
Go report was complaining that we didn't comment these exported types, should be fixed now though